nilearn/_utils/cache_mixin.py

```diff
@@ -108,11 +108,11 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
     memory: instance of joblib.Memory or string
         Used to cache the function call.
 
-    func_memory_level: int, optional
+    func_memory_level: int
```

Comment:
Looks like you meant to make both `func_memory_level` and `memory_level` parameters required but didn't do it in the function signature.

Also just wondering whether something more explicit than func_memory_level could be used ... No great suggestions other than reference_memory_level though.